SCClassLibrary/Common/Control/Server.sc

```diff
@@ -171,7 +232,7 @@ ServerOptions {
 		^this.primitiveFailed
 	}
 
-	numPrivateAudioBusChannels_ { |numChannels = 112|
+	numPrivateAudioBusChannels_ { |numChannels = 1020|
```

Comment:
```suggestion
	numPrivateAudioBusChannels_ { |numChannels = (defaultValues[\numPrivateAudioBusChannels])|
```

Reply:
I didn't know you could use instance variables in this way. I'm guessing the `( )` are necessary?

Reply:
Unfortunately, this doesn't seem to work. When writing `s.options.numInputBusChannels_( `, the tooltip shows only the argument name `numChannels`. The method documentation also shows only the argument name.